ktor-server/ktor-server-core/jvmAndNix/src/io/ktor/server/request/PipelineRequest.kt

```diff
@@ -97,3 +97,23 @@ public fun ApplicationRequest.encodeParameters(parameters: Parameters): Paramete
         }
     }.build()
 }
+
+/**
+ * Converts parameters to query parameters by fixing the [Parameters.get] method
+ * to make it return an empty string for the query parameter without value
+ */
+public fun Parameters.toQueryParameters(): Parameters {
```

Comment:
Could we make it internal?

Reply:
Do you mean with the internal annotation?

Reply:
nope, visibility. I would rather copy the method if needed to keep the fix for the patch release

Reply:
I don't think it's a good idea because it would require copying to 4 different modules.